tests/migrations/test_runner.py

```diff
@@ -64,6 +64,10 @@ def test_version() -> None:
 
 def test_no_schema_differences() -> None:
     storages_to_test = [
+        StorageKey.EVENTS,
```

Comment:
Now I get why this test was passing even after I broke the schema with the tags_hash_map.